tests/virt_autotest/update_package.pm

```diff
@@ -48,8 +48,8 @@ sub update_package {
 sub run {
     my $self = shift;
     #workaroud: skip update package for registered aarch64 tests and because there are conflicts on sles15sp2 XEN
-    $self->update_package() unless (is_registered_sles && is_aarch64);
-    unless ((is_registered_sles && is_aarch64) || is_s390x) {
+    $self->update_package() unless (!!get_var('AUTOYAST') || is_registered_sles && is_aarch64);
```

Comment:
Why use two exclamations ? This can be improved I think.

Reply:
https://stackoverflow.com/questions/2168406/what-does-double-exclamation-point-mean

There are some usage examples in openqa source.  you can grep them.

Reply:
It seems ```!!``` will evaluate
```
everything greater than 0 to 1
everything smaller than 0 to 1
0 to empty
any string to 1
```
Is this what you want ?

Reply:
It converts the value to a boolean.

Reply:
You can print the result out, it converts 0 to empyt, and
```
everything greater than 0 to 1
everything smaller than 0 to 1
any string to 1
```